app/router.js

```diff
@@ -8,6 +8,9 @@ const Router = Ember.Router.extend({
 Router.map(function() {
   this.route('about');
   this.route('contact');
+  this.route('rentals', function() {
+    this.route('show', { path: '/:title' });
```

Comment:
@sbatson5 Let's just add it to this one, makes it quite clear why we're adding it :-)